common/djangoapps/student/views/dashboard.py

```diff
@@ -848,7 +863,22 @@ def student_dashboard(request):  # lint-amnesty, pylint: disable=too-many-statem
         'resume_button_urls': resume_button_urls
     })
 
-    response = render_to_response('dashboard.html', context)
+    dashboard_template = 'dashboard.html'
+    try:
+        # .. filter_implemented_name: DashboardRenderStarted
+        # .. filter_type: org.openedx.learning.dashboard.render.started.v1
+        context, dashboard_template = DashboardRenderStarted.run_filter(
+            context=context, template_name=dashboard_template,
+        )
+    except DashboardRenderStarted.RenderAlternativeDashboard as exc:
+        response = render_to_response(exc.dashboard_template, exc.template_context)
+    except DashboardRenderStarted.RedirectToPage as exc:
+        response = HttpResponseRedirect(exc.redirect_to or 'account_settings')
```

Comment:
`<lms_root>/account_settings` 404s.. shouldn't this be:

```suggestion
        response = HttpResponseRedirect(exc.redirect_to or reverse('account_settings'))
```

Reply:
I absolutely missed this. Thank you so much!